src/modules/App/stories/integrated.stories.js

```diff
@@ -378,7 +440,75 @@ export const GroupChannel = () => {
                   // CHANNEL_SETTING__MODERATION__EMPTY_BAN: '차단된 된 유저가 아무도 없습니다',
                   // CHANNEL_SETTING__MODERATION__ALL_BAN: '차단된 유저 모두보기'
                 }}
-              />
+              >
+                <div style={{height: '100%', width: '100%', display: 'flex', flexDirection: 'row'}}>
+                  <GroupChannelList
+                    selectedChannelUrl={channelUrl}
+                    onChannelSelect={(channel) => setChannelUrl(channel?.url ?? '')}
+                    onCreateChannel={(channel) => setChannelUrl(channel.url)}
+                  />
+                  <div style={{height: '100%', width: '100%', display: 'inline-flex', flexDirection: 'row'}}>
+                    <div style={{width: '100%'}}>
+                      <Channel
+                        channelUrl={channelUrl}
+                        renderSuggestedReplies={(suggestedRepliesProps) => {
+                          const { replyOptions, onSendMessage, message } = suggestedRepliesProps;
+                          return <DraggableSuggestedReplies options={replyOptions} />;
+                        }}
+                      />
+                    </div>
+                  </div>
+                </div>
+              </SendbirdProvider>
+              // <App
```

Comment:
You will might remove this comment before merging :D

Reply:
yes ill put this file back to before